samples/apps/bookinfo/rules/destination-policy-reviews.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: config.istio.io/v1beta1
+kind: DestinationPolicy
+metadata:
+  name: reviews
+spec:
+  policy:
```

Comment:
Let's use the new format and deprecate the policy list. The code is already implemented in pilot PR.

Reply:
OK, I'll change it